src/main/webapp/Portal.html

```diff
@@ -485,6 +485,10 @@
         srcPath = cdnEndpoint + 'generated/synapse-react-client.development.js'
       }
 
+      // TODO: Workaround for SWC-6664
+      // Override the conditional above and just use the production distribution
+      reactQueryPath = cdnEndpoint + 'generated/react-query.production.min.js'
```

Comment:
Please remove the assignments to reactQueryPath above this one (ln 478 and 484) for clarity.